build.gradle

```diff
@@ -426,6 +431,39 @@ task webManual(type: Sync) {
     }
 }
 
+task genMac {
+    description = 'Generate the Mac .app skeleton. Depends AppBundler (https://github.com/TheInfiniteKind/appbundler)'
+    outputs.dir file("${buildDir}/appbundler")
+    doLast {
+        ant.taskdef(name: 'appbundler',
+                classname: 'com.oracle.appbundler.AppBundlerTask',
+                classpath: configurations.genMac.asPath)
+        ant.appbundler(outputdirectory: "${buildDir}/appbundler",
+                name: applicationName,
+                displayname: applicationName,
+                executablename: applicationName,
+                identifier: 'org.omegat.OmegaT',
+                icon: 'images/OmegaT.icns',
+                version: '${version}',
+                jvmrequired: javaVersion,
+                shortversion: '${version}',
+                mainclassname: mainClassName) {
+            option(value: "-Xdock:name=${applicationName}")
+            option(value: "-Dapple.awt.application.name=${applicationName}")
+            option(value: "-Dapple.awt.application.appearance=system")
+            argument(value: '--config-file=${configfile}')
```

Comment:
It seems not working. There is no definition "configfile" and it does not replace with variable because string has single quote.